src/reference/modules/usage/pages/configuration_management.adoc

```diff
@@ -134,6 +137,15 @@ The information about a Node's policy server.
 * `${rudder.node.policyserver.hostname}` returns the hostname of the Policy Server
 * `${rudder.node.policyserver.admin}` returns the administrator login of the Policy Server
 
+==== Node-level system properties
+
+These properties are evaluated on the node at run time, and are hence available both in
+directives parameters and in the technique editor:
+
+* `${sys.fqhost}` is the fully-qualified hostname, it matches the hostname seen in Rudder
+* `${sys.uqhost}` is the unqualified hostname
+* `${sys.host}` is the node's hostname (according to the kernel)
+* `${sys.domain}` is the domain discovered by the agent
```

Comment:
Would you mind to add a link toward some place where other such properties are documented, even if it's outside of Rudder?

Reply:
Good idea!